core/domain/question_services.py

```diff
@@ -142,6 +142,26 @@ def get_questions_and_skill_descriptions_by_skill_ids(
     return questions, skill_descriptions, next_cursor
 
 
+def get_questions_by_skill_ids(question_count, skill_ids):
```

Comment:
Any max limits on question_count (to prevent system overload)? If so, enforce them here. You can have tunable constants like MAX_QUESTIONS_FETCHABLE_AT_ONE_TIME in feconf.py (or assets/constants.js if you need that info in the frontend).

Reply:
Done.